profiles/epub.md

```diff
@@ -1,10 +1,24 @@
 # EPUB Profile
 
+## Introduction
+
 While EPUB 2.x and 3.x can mostly be mapped directly to the Readium Web Publication Manifest, a number of metadata, collection roles and properties are still very specific to EPUB and not fully covered by the core specification.
 
-Thanks to the various extension points in place, this document defines a number of new collection roles and properties that are for the most part exclusive to EPUB.
+This profile relies on:
+
+* the definition of new [collection roles](#2-collection-roles),
+* the definition of new [resource properties](#3-resource-properties),
+* the [encryption module](../modules/encryption.md).
+
+## 1. Declaring conformance with the EPUB Profile
+
+In order to declare that it conforms to the EPUB Profile, a Readium Web Publication Manifest <strong class="rfc">must</strong> include a `conformsTo` key in its `metadata` section, with `https://readium.org/webpub-manifest/profiles/epub` as value.
 
-## Collection Roles
+## 2. Restrictions on resources in the `readingOrder`
+
+A Readium Web Publication Manifest that conforms to the EPUB Profile <strong class="rfc">must</strong> strictly reference XHTML documents (`application/xhtml+xml`) in its `readingOrder`.
```

Comment:
These two things are currently missing in mobile toolkits.

* `conformsTo` is new of course, I'll wait for it to be fully merged in the spec before implementing it.
* We don't enforce `application/xhtml+xml` in the reading order of an EPUB. Should we remove non XHTML resources from the reading order, or refuse the EPUB as being invalid, or don't check the media types declared in the OPF at all, as right now?

Reply:
I'm not in favor of validating this rule in the User Agent (either by removing non xhtml resources or by refusing the publication). The User Agent should do as much as it can to display the content. But if the resource is a source of problems for the User Agent, then yes it should be removed from the reading order (example = a PDF inside the reading order of an EPUB).